api/metrics-api-jaxrs/pom.xml

```diff
@@ -161,172 +167,42 @@
             -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=8787</jvmArgs>
         </configuration>
       </plugin>
-
     </plugins>
   </build>
 
   <profiles>
     <profile>
-      <id>REST doc generation</id>
-      <activation>
-        <property>
-          <name>docgen</name>
-        </property>
-      </activation>
-
-      <dependencies>
-        <dependency>
-          <groupId>org.rhq.helpers</groupId>
-          <artifactId>rest-docs-generator</artifactId>
-          <version>${rest-docs-generator.version}</version>
-          <scope>provided</scope>
-        </dependency>
-      </dependencies>
+      <id>docgen</id>
 
```

Comment:
I'm fine with activating a profile directly with -P instead of indirectly with -D

Maybe it's not a shared practice in all Hawkular projects though. Do you know what others do?